src/main/java/Contacts1.java

```diff
@@ -593,8 +593,16 @@ private static boolean isValidName(String name) {
      * @param phone to be validated
      */
     private static boolean isValidPhone(String phone) {
-        return !phone.isEmpty();
-        //TODO: implement a better validation
+        if (phone.isEmpty()){
+            return false;
+        }
+
+        try {
+            int phoneNumber = Integer.parseInt(phone);
+            return (phoneNumber > 79999999 && phoneNumber < 100000000);
```

Comment:
Some phone numbers, particularly landlines, do start with 6 so I would change this a bit.

Reply:
I updated it! Though I'm not sure if the try-catch thing is the right way to deal with non integer phone numbers or if there's a better way